src/epub/text/endnotes.xhtml

```diff
@@ -379,7 +379,7 @@
Scotland, Ireland, and Wales. For France, <abbr epub:type="z3998:given-name">P.</abbr> Viollet, <i epub:type="se:name.publication.book" xml:lang="fr">Précis de l’histoire du droit français: Droit privé</i>, 1886, and several of his monographs in <i epub:type="se:name.publication.book" xml:lang="fr"><abbr>Bibl.</abbr> de l’École des Chartes</i>; Babeau, <i epub:type="se:name.publication.book" xml:lang="fr">Le Village sous l’ancien régime</i> (the <i xml:lang="ru-Latn">mir</i> in the eighteenth century), third edition, 1887; Bonnemère, Doniol, <abbr class="eoc">etc.</abbr> For Italy and Scandinavia, the chief works are named in Laveleye’s <i epub:type="se:name.publication.book">Primitive Property</i>, German version by <abbr epub:type="z3998:given-name">K.</abbr> Bücher. For the Finns, Rein’s <i epub:type="se:name.publication.book" xml:lang="de">Föreläsningar</i>, <span epub:type="z3998:roman">i</span>, 16; Koskinen, <i epub:type="se:name.publication.book" xml:lang="de">Finnische Geschichte</i>, 1874, and various monographs. For the Lives and Coures, <abbr epub:type="z3998:name-title">Prof.</abbr> Lutchitzky in <i epub:type="se:name.publication.book" xml:lang="ru-Latn">Severnyi Vestnik</i>, 1891. For the Teutons, besides the well-known works of Maurer, Sohm (<i epub:type="se:name.publication.book" xml:lang="de">Altdeutsche Reichs-und Gerichts-Verfassung</i>), also Dahn (<i epub:type="se:name.publication.book" xml:lang="de">Urzeit, Völkerwanderung, Langobardische Studien</i>), Janssen, Wilh. Arnold, <abbr class="eoc">etc.</abbr> For India, besides <abbr epub:type="z3998:given-name">H.</abbr> Maine and the works he names, Sir John Phear’s <i epub:type="se:name.publication.book">Aryan Village</i>. For Russia and South Slavonians, see Kavelin, Posnikoff, Sokolovsky, Kovalevsky, Efimenko, Ivanisheff, Klaus, <abbr>etc.</abbr> (copious bibliographical index up to 1880 in the <i epub:type="se:name.publication.book" xml:lang="ru-Latn">Sbornik svedeniy ob obschinye</i> of the <abbr>Russ.</abbr> <abbr>Geog.</abbr> <abbr>Soc.</abbr>). For general conclusions, besides Laveleye’s <i epub:type="se:name.publication.book" xml:lang="fr">Propriété</i>, Morgan’s <i epub:type="se:name.publication.book">Ancient Society</i>, Lippert’s <i epub:type="se:name.publication.book" xml:lang="de">Kulturgeschichte</i>, Post, Dargun, <abbr>etc.</abbr>, also the lectures of <abbr epub:type="z3998:given-name">M.</abbr> Kovalevsky (<i epub:type="se:name.publication.book" xml:lang="fr">Tableau des origines et de l’evolution de la famille et de la propriété</i>, Stockholm, 1890). Many special monographs ought to be mentioned; their titles may be found in the excellent lists given by <abbr epub:type="z3998:given-name">P.</abbr> Viollet in <i epub:type="se:name.publication.book" xml:lang="fr">Droit privé</i> and <i epub:type="se:name.publication.book" xml:lang="fr">Droit public</i>. For other races, see subsequent notes. <a href="chapter-4.xhtml#noteref-117" epub:type="backlink">↩</a></p>
```

Comment:
I think the space needs to remain after Richts- because it's meant to connect with -Verfassung

Reply:
OK, that should be fixed now.